app/views/roles/_form.html.erb

```diff
@@ -10,8 +10,14 @@
 
   <div class="tab-content">
     <div class="tab-pane active" id="primary">
+      <% if @role.cloned_from_id.present? && User.current.can?(:view_roles, @role.cloned_from) %>
```

Comment:
@rohoover I think the mix of styles used here isn't quite good, it looks inconsistent (2 styles to show an info message). Not sure how to fix it, any ideas?

![screenshot from 2017-03-08 10-32-18](https://cloud.githubusercontent.com/assets/598891/23698098/872b9202-03ea-11e7-9d3d-940112bc8d43.png)

@ares I'm happy to merge and fix it later.